api/models/resource.util.js

```diff
@@ -199,6 +205,16 @@ exports.setOrganizations = async (resource, orgs) => {
   );
 };
 
+exports.setUser = async (resource, user) => {
+  return await queries.execUpdateSetManyToMany(
+    Resource,
+    'resources',
+    resource,
+    User,
+    'user',
+    [user]
+  );
+};
```

Comment:
This isn't many to many, it's many (resource) to one (user). You'll want to use `execUpdateSetManyToOne`.